src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinEnchantment_FixRomanNumerals.java

```diff
@@ -0,0 +1,31 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import com.mitchej123.hodgepodge.Common;
+import com.mitchej123.hodgepodge.util.RomanNumerals;
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.util.StatCollector;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+/**
+ * Mixin from Sk1erLLC/Patcher
+ */
+@Mixin(Enchantment.class)
+public abstract class MixinEnchantment_FixRomanNumerals {
```

Comment:
Doing this via mixins is unreasonable. MC is already able to properly display the numbers for enchantments >4, they are just not localized. Just add the higher numbers to the en_US.lang file:
```
enchantment.level.4=V
enchantment.level.5=VI
enchantment.level.6=VII
enchantment.level.7=VIII
enchantment.level.8=IX
enchantment.level.9=X
etc.
```

Reply:
They are already in the lang file. The fix makes it work for any enchantment level, so above 10,
. And I need a mixin anyways for the feature that displays enchantments levels as arabic numbers instead if roman numbers 
![image](https://user-images.githubusercontent.com/57050655/210876911-53229053-7580-4a13-9bd0-836d00849ddb.png)

Reply:
>Then just add the localizations for >10...

I am not going to write them manually. Can you even modifiy the vanilla lang file ? And how does that work if you have another language selected ?

>The way you currently do it is a breaking change. 

And what does it break exactly ? It covers more cases than before and handles the same way the cases that vanilla could handle before

Reply:
> Can you even modifiy the vanilla lang file?

First of all, you are not modifying it, you are extending it - which works the same as with any other localization. Sencond, of course you can modify vanilla localizations, resource packs are doing this for years.
> And what does it break exactly ? It covers more cases than before and handles the same way the cases that vanilla could handle before

You are *not* handling it the same way as vanilla, you are doing it completely different. While doing so, you are removing the ability to localize the numbers, which *could already* be used to replace them arabic numbers. Hodgepodge is intended to be usable outside of GTNH and you don't know what the entirety of 1.7.10 mods / resourcepacks expects. So *extending* is better than *replacing* in this case.

Reply:
just a question: if we add new lang keys, that means no ressource pack on earth have them no? so how is that a breaking change? And what's the maximum enchant one can do via MC? because i feel like it's going to be a non neglectible amount of new lang files just for a really small benefit.

Reply:
Enchantments can have a max amplifier of 128 (0-127, it's stored as short in the ItemStack's NBT).
My biggest issue with the new implementation is that it *replaces* the old system entirely, which used localizations for the first 10 levels. So anything that relied on that won't work afterwards. I know that at least one mod extends this to 50, sadly I don't know which one anymore, so I can't say in howfar it relied on the current system.